src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1407,7 +1407,7 @@ def _configure_security_policy_storage_params(arg_ctx):
         c.argument('family',
                    arg_type=family_param_type,
                    help='The compute generation component of the sku. '
-                   'Allowed values include: Gen4, Gen5.')
+                   'Allowed values include: Gen5.')
```

Comment:
If allowed value changes, it will cause breaking change to users. Could you please confirm the breaking change here?

Reply:
Thanks for bringing this up. I've synced with my PM and we decided to revert the change.